include/zephyr/drivers/sensor_v2.h

```diff
@@ -0,0 +1,989 @@
+/*
+ * Copyright (c) 2022 Google Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file drivers/sensor_v2.h
+ *
+ * @brief Public APIs for the sensor driver.
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_SENSOR_V2_H_
+#define ZEPHYR_INCLUDE_DRIVERS_SENSOR_V2_H_
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_SENSOR_H_
+#error "Please include drivers/sensor.h directly"
```

Comment:
Nit: If somebody did this by accident, a more helpful message might be, "Do not include drivers/sensor_v2.h directly."